#!/bin/sh -e
##  by Joost Yervante Damad <joost@lumatec.be>
##
## DP: use -O1 for m68k

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
--- openmsx-0.6.1.orig/build/cpu-m68k.mk	2005-06-25 09:41:53.000000000 +0200
+++ openmsx-0.6.1/build/cpu-m68k.mk	2006-02-04 15:06:39.000000000 +0100
@@ -8,3 +8,5 @@
 # Default build flavour.
 OPENMSX_FLAVOUR?=opt
 
+# overrule optimization level for m68k
+CXXFLAGS:=-O1 -DNDEBUG

